src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEventNotificationProcessingServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.util.Collection;
+
+import org.kuali.rice.krad.util.ObjectUtils;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ExpenseDetailedReportDTO;
+import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationService;
+
+public class ConcurEventNotificationProcessingServiceImpl implements ConcurEventNotificationProcessingService {
+    
+    protected ConcurEventNotificationService concurEventNotificationService;
+    protected ConcurAccountValidationService concurAccountValidationService;
+
+    @Override
+    public void processConcurEventNotifications() {
+        Collection<ConcurEventNotification> concurEventNotifications = concurEventNotificationService.retrieveConcurEventNotificationsForProcessing();
+
+        if (ObjectUtils.isNotNull(concurEventNotifications)) {
+            for (ConcurEventNotification concurEventNotification : concurEventNotifications) {
+                ExpenseDetailedReportDTO expenseDetailedReport = retrieveExpenseDetailedReportFromConcur(concurEventNotification);
+                ConcurAccountInfo concurAccountInfo = extractAccountInfo(expenseDetailedReport);
+                concurAccountValidationService.validateConcurAccountInfo(concurAccountInfo);
+                updateExpenseReportStatusInConcur();
+            }
+        }
+    }
+
+    @Override
+    public ExpenseDetailedReportDTO retrieveExpenseDetailedReportFromConcur(ConcurEventNotification concurEventNotification) {
+        //TODO add implementation
```

Comment:
FWIW, I don't like TODO comments. I would much prefer having a JIRA for the future work and leaving the comment out of the code.